src/fr/articles/liens-telechargement.md

```diff
@@ -11,28 +11,28 @@ tags:
 
 Il existe différents types de liens : [les liens classiques](/fr/web/developper/navigation-generale/) permettant d'atteindre une nouvelle page, [les liens d'évitement](/fr/articles/liens-evitement/) permettant de se déplacer au sein de la page et enfin ceux qui nous intéressent dans cet article : les liens permettant de télécharger un fichier.
 
-Un lien permettant de télécharger un fichier devrait respecter les règles suivantes :
+Un lien permettant de télécharger un fichier devrait, à la fois , pour des raisons d'accessibilité et d'ergonomie, respecter les règles suivantes :
```

Comment:
```suggestion
Un lien permettant de télécharger un fichier devrait, à la fois, pour des raisons d'accessibilité et d'ergonomie, respecter les règles suivantes :
```

Reply:
Maybe something like 'Pour des raisons d'accessibilité et d'ergonomie, un lien de téléchargement devrait respecter les règles suivantes :' would be shorter and clearer?